src/libertem/api.py

```diff
@@ -517,8 +618,9 @@ def map(self, dataset, f, roi=None):
         Returns
         -------
 
-        BufferWrapper:
-            The result of the UDF. Access the underlying numpy array using the `data` attribute.
+        BufferWrapper : libertem.common.buffers.BufferWrapper
+            The result of the UDF. Access the underlying numpy array using the
+            :attr:`~libertem.common.buffers.BufferWrapper.data` property.
```

Comment:
... maybe we should also note that a `BufferWrapper` can just be passed to `numpy` functions in place of any array (also applies to other `BufferWrapper` mentions)